### PR TITLE
[Feature][Connector] Re-add support for elasticsearch 6.x in source code.

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Purpose of this pull request
Re-add support for elasticsearch 6.x in source code. So user can package again to support elasticsearch 6.x. 
The next step will be to generate separate jar plugins for the two versions separately

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/development/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-elasticsearch7/pom.xml [hunk 1]
```diff
@@ -27,7 +27,11 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>seatunnel-connector-flink-elasticsearch</artifactId>
+    <artifactId>seatunnel-connector-flink-elasticsearch7</artifactId>
+
+    <properties>
+        <elasticsearch7.version>7.5.1</elasticsearch7.version>
+    </properties>
```
Comment (CalvinKirs) [inline seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-elasticsearch7/pom.xml]: The version number needs to be declared in the root-pom

Comment (Hisoka-X) [reply seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-elasticsearch7/pom.xml]: Done

File: docs/en/connector/sink/Elasticsearch.mdx [hunk 2]
```diff
@@ -12,7 +12,7 @@ Output data to `Elasticsearch`.
 Engine Supported and plugin name
 
 * [x] Spark: Elasticsearch(supported `ElasticSearch version is >= 2.x and <7.0.0`)
-* [x] Flink: Elasticsearch(supported `ElasticSearch version is >= 2.x and <8.0.0`)
+* [x] Flink: Elasticsearch(supported `ElasticSearch version = 7.x`)
```
Comment (CalvinKirs) [inline docs/en/connector/sink/Elasticsearch.mdx]: Do we need to state here, if users use 6, they can pack it themselves (and the specific steps)

Comment (BenJFan) [reply docs/en/connector/sink/Elasticsearch.mdx]: Done

Comment (Hisoka-X) [reply docs/en/connector/sink/Elasticsearch.mdx]: Done

**Review by CalvinKirs — APPROVED** [review]

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Pull request merged**